request/request.go

```diff
@@ -226,19 +226,15 @@ func (r *Request) SizeAndDo(m *dns.Msg) bool {
 	return true
 }
 
-// Scrub is a noop function, added for backwards compatibility reasons. The original Scrub is now called
-// automatically by the server on writing the reply. See ScrubWriter.
```

Comment:
Was this comment incorrect? I mean, was the real Scrub never getting called in this case? So, we're not calling it twice now are we?

Reply:
yes, we we're calling the noop function.